### DISCUSSION TITLE
jvm crach, add jvm_args: -javaagent:/home/app/apm-agent-8.8/skywalking-agent.jar

### DISCUSSION THREAD

**Discussion by wzhangNJ** [apache/skywalking#8648] [Q&A]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar issues.

### Apache SkyWalking Component

Java Agent (apache/skywalking-java)

### What happened

add VM Arguments: jvm_args: -javaagent:/data/agent/skywalking-agent.jar occur jvm crach!
SkyWalking version： 8.8.0
OS version ：  Linux version 3.10.0-693.2.2.el7.x86_64 (builder@kbuilder.dev.centos.org) (gcc version 4.8.5 20150623 (Red Hat 4.8.5-16) (GCC) 
JRE version ：1.8.0_161

<img width="479" alt="image" src="https://user-images.githubusercontent.com/19465278/157386272-e1396dbb-6c1d-4e16-81e7-6ec742ef0e1f.png">

### What you expected to happen

We need help！

### How to reproduce

Occasional appearance

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by wu-sheng** [comment]
Hi, this kind of report seems not very useful. You should know a community is just helping you if possible, not have to. You could do some research and find out why first.

**Reply by wzhangNJ** [reply]
Thank you，Mr wu!
We are also investigating the cause of the problem,I also saw feedback the issue in the community.I also read your answer.
I also want to solve this problem.Our company is using it(400+App，20T data per day).
We do encounter some problems in the process of use,We want to work with the community!
